rfcs/0153-uefi-only.md

```diff
@@ -0,0 +1,174 @@
+---
+feature: non_legacy_boot
+start-date: 2023-06-16
+author: Ryan Lahfa
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+NixOS will have first-class support for UEFI
+and uses it as a default boot environment, for supported architectures,
+even in situations where only BIOS Boot Specification's legacy boot is available,
+via a dual-stage payload consisting of a polyfill bootloader/firmware and a standard
```

Comment:
Please no.  If you want these Rube Goldberg contraptions on your machine, great.  Don't force them on everybody else.

Reply:
I stopped reading at this point.